src/ConfigServiceProvider.php

```diff
@@ -4,7 +4,10 @@
 
 use League\Container\ServiceProvider\AbstractServiceProvider;
 use League\Container\ServiceProvider\BootableServiceProviderInterface;
-use TomPHP\ConfigServiceProvider\Exception\NoMatchingFilesException;
+use League\Container\ServiceProvider\ServiceProviderInterface;
+use TomPHP\ConfigServiceProvider\Exception\EntryDoesNotExistException;
```

Comment:
@localheinz Strange - I thought `php-cs-fixer` was picking these up, will check again. (and will fix)